### DISCUSSION TITLE
Version 0.84.18 ?

### DISCUSSION THREAD

**Discussion by ninjagang** [nocodb/nocodb#1852] [General]
I have Installed version 0.90.7，But it still display version : 0.84.18

![ver](https://user-images.githubusercontent.com/34408821/164965130-9593768f-71e3-4674-aa86-eb34851d44a2.png)

**Comment by wingkwong** [comment]
1. Try Hard Refresh / or verify in other browser
2. Make sure that you've installed 0.90.7. If you were using docker with latest tag, please remove the local image first and pull it again.

**Comment by o1lab** [comment]
Moving this to discussion

**Comment by ninjagang** [comment]
finally, i  upgraded nocodb to 0.90.7。
step 1(0.84.18):
`npm install`

setp 2:(0.90.7)
`npm uninstall`
`npm install --save nocodb`
